change/@ni-nimble-components-8010f392-806a-4399-bf80-9fcd41f97cb5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Since this doesn't affect the bits of the built package (only the tests) the `type` and `dependentChangeType` can both be `none`.

Reply:
applied